places/sandiego.md

```diff
@@ -0,0 +1,3 @@
+## San Diego, CA
+### Great Places To Eat
```

Comment:
I actually do not know ... I've only had it catered ... never been to an Indian restaurant here!